sunkit_pyvista/tests/test_plotting.py

```diff
@@ -48,6 +48,12 @@ def test_plot_map_with_functionality(aia171_test_map, plotter, verify_cache_imag
                           obstime=aia171_test_map.date)
     plotter.plot_coordinates(coordinate, color='blue')
 
+    filepath = (tmp_path / "save_data.vtm")
```

Comment:
Does this also test the colormap works properly now?

Reply:
Yes, I could update the test for saving to include a map, I'll add that in actually.